### PR TITLE
Add documentation for Micrometer OpenTelemetry registry

### PR DESCRIPTION
**Pull request by serg-firsov**
This documentation is for 10.17.0

### CODE REVIEW

File: content/en/docs/refguide/runtime/metrics.md [hunk 1]
```diff
@@ -40,32 +40,37 @@ Micrometer can send metrics to multiple registries. To configure micrometer for
 
 The details of each settings are listed below.
 
-* `type` *(mandatory)* – the type of registry to use. Currently supported types are [`prometheus`](https://prometheus.io/docs/introduction/overview/), [`jmx`](https://www.oracle.com/java/technologies/javase/javamanagement.html), [`influx`](https://www.influxdata.com/), and [`statsd`](https://www.datadoghq.com/dg/monitor/ts/statsd/). Depending on the type of the registry the `settings` may vary.
+* `type` *(mandatory)* – the type of registry to use. Currently supported types are [`prometheus`](https://prometheus.io/docs/introduction/overview/), [`jmx`](https://www.oracle.com/java/technologies/javase/javamanagement.html), [`influx`](https://www.influxdata.com/), [`statsd`](https://www.datadoghq.com/dg/monitor/ts/statsd/) and [`opentelemetry`](https://opentelemetry.io/docs/). Depending on the type of the registry the `settings` may vary.
```
Comment (atennapel) [inline content/en/docs/refguide/runtime/metrics.md]: I think the docs team employs the "Oxford comma" convention, meaning there should be a comma before the "and".

File: content/en/docs/refguide/runtime/metrics.md [hunk 2]
```diff
@@ -214,6 +219,46 @@ Example 2
 ]
 ```
 
+#### OpenTelemetry{#opentelemetry}
```
Comment (atennapel) [inline content/en/docs/refguide/runtime/metrics.md:222]: I think we'll need a warning here that this was introduced in 10.17.0.

Comment (atennapel) [reply content/en/docs/refguide/runtime/metrics.md:222]: Like this:
```
{{% alert color="info" %}}
This feature was introduced in Mendix versions 10.17.0.
{{% /alert %}}
```

**Review by atennapel — CHANGES_REQUESTED** [review]

File: content/en/docs/refguide/runtime/metrics.md (hunk 1 above)
Comment (serg-firsov) [reply content/en/docs/refguide/runtime/metrics.md]: Done

File: content/en/docs/refguide/runtime/metrics.md (hunk 2 above)
Comment (serg-firsov) [reply content/en/docs/refguide/runtime/metrics.md:222]: Done

File: content/en/docs/refguide/runtime/metrics.md (hunk 1 above)
Comment (atennapel) [reply content/en/docs/refguide/runtime/metrics.md]: Approved.

File: content/en/docs/refguide/runtime/metrics.md (hunk 2 above)
Comment (atennapel) [reply content/en/docs/refguide/runtime/metrics.md:222]: Approved.

**Review by atennapel — APPROVED** [review]

**Comment by atennapel** [comment]
Technical review done, approved.

**Review by MarkvanMents — APPROVED** [review]
Thanks for the PR. Have done a very light proofread here: https://github.com/mendix/docs/pull/8555/commits/92b881316ea6221393a0f880fc3881d91b4b7703
Approving

**Pull request merged**